PSDepend/PSDependScripts/GitHub.ps1

```diff
@@ -278,13 +328,13 @@ if($ShouldInstall)
     {
         # Repository does not seem to exist or a branch is the target
         $ShouldInstall = $False
+        Write-Warning "Could not find module on GitHub: $_"
```

Comment:
I've updated the comment two lines above in my previous commit. This case can occur when using branch names instead of version numbers too. Thus, I'd say the warning is not entirely correct.